Concurrent file access in add lifetime Tests

Motivation
----------
As found by @rcarpa, the `test_add_lifetime_exception` and `test_add_lifetime_exception_large_dids_number` use the same temporary file for some internal behavior.

This sometimes creates nondeterministic errors in the tests.

```text
=================================== FAILURES ===================================
___________________ TestBinRucio.test_add_lifetime_exception ___________________
[gw0] linux -- Python 3.9.9 /usr/bin/python
test_bin_rucio.py:2176: in test_add_lifetime_exception
    assert "One or more DIDs are containers" in err
E   assert 'One or more DIDs are containers' in '\x1b[31;1m2022-06-13 07:23:46,503\tERROR\t[Errno 2] No such file or directory: \'/tmp/rootlifetime_exception.txt\'\x1b[0m\n\x1b[31;1m2022-06-13 07:23:46,505\tERROR\t\nRucio exited with an unexpected/unknown error.\nPlease rerun the last command with the "-v" option to gather more information.\nIf it\'s a problem concerning your experiment or if you\'re unsure what to do, please follow up at: rucio-dev@cern.ch\nIf you\'re sure there is a problem with Rucio itself, please follow up at: [https://github.com/rucio/rucio/issues/\x1b[0m\n](https://github.com/rucio/rucio/issues//x1b[0m/n)'
----------------------------- Captured stdout call -----------------------------
rucio add-lifetime-exception --inputfile /tmp/rootlifetime_exception.txt --reason "Needed for analysis" --expiration 2015-10-30
0  2022-06-13 07:23:44,539	WARNING	data13_hip:dataset_eeca70c7d199424e8cb2ae29e9735f88 is not affected by the lifetime model
2022-06-13 07:23:44,540	ERROR	Nothing to submit
rucio add-lifetime-exception --inputfile /tmp/rootlifetime_exception.txt --reason "Needed for analysis" --expiration 2015-10-30
1  2022-06-13 07:23:46,503	ERROR	[Errno 2] No such file or directory: '/tmp/rootlifetime_exception.txt'
2022-06-13 07:23:46,505	ERROR	
Rucio exited with an unexpected/unknown error.
Please rerun the last command with the "-v" option to gather more information.
If it's a problem concerning your experiment or if you're unsure what to do, please follow up at: rucio-dev@cern.ch
If you're sure there is a problem with Rucio itself, please follow up at: https://github.com/rucio/rucio/issues/
```

Got introduced in #5481.

Modification
------------
Use separate files for each test.

--- a/lib/rucio/tests/test_bin_rucio.py
+++ b/lib/rucio/tests/test_bin_rucio.py
@@ -2183,7 +2183,7 @@ class TestBinRucio(unittest.TestCase):
 
     def test_add_lifetime_exception_large_dids_number(self):
         """ CLIENT(USER): Check that exceptions with more than 1k DIDs are supported """
-        filename = get_tmp_dir() + 'lifetime_exception.txt'
+        filename = get_tmp_dir() + 'lifetime_exception_many_dids.txt'
         with open(filename, 'w') as file_:
             for _ in range(2000):
                 file_.write('%s:%s\n' % (self.user, generate_uuid()))

